voice/handle-user-input.py

```diff
@@ -26,7 +26,7 @@ def answer_call():
 
 
 @app.route("/webhooks/dtmf", methods=['POST'])
-def inout_webhook():
+def input_webhook():
```

Comment:
Didn't need fixing, but if we are going to change it to something we should probably change the function name to `dtmf`

Reply:
I did this change based on the other language sample code `NodeJs`, 'Java'.
I felt inout_webhook was a type.

Does this change even require?